packages/components/atoms/Typography/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@no-gravity-elements/typography",
-  "version": "1.0.0",
+  "version": "1.0.8",
```

Comment:
Can we delete versions from 1.0.1 to 1.0.8 in the npm registry? so next time we push this guy it will have the correct version

Reply:
I don't follow what you mean exactly. You mean delete all the versions in the registry and roll back to 1.0.0?

Reply:
yes, rollback to version 1.0.0 or 1.0.1. Since this library won't be used only internally like the tools repo, it's better to have it as clean as possible